crypto/x509v3/pcy_cache.c

```diff
@@ -26,21 +26,25 @@ static int policy_cache_set_int(long *out, ASN1_INTEGER *value);
 static int policy_cache_create(X509 *x,
                                CERTIFICATEPOLICIES *policies, int crit)
 {
-    int i;
-    int ret = 0;
+    int i, ret = 0;
     X509_POLICY_CACHE *cache = x->policy_cache;
     X509_POLICY_DATA *data = NULL;
     POLICYINFO *policy;
+
     if (sk_POLICYINFO_num(policies) == 0)
-        goto bad_policy;
+        goto fail_policy;
```

Comment:
I think this should be reverted to `bad_policy`, because it's a wrong argument provided by the caller.